utilities/transcript.test.ts

```diff
@@ -0,0 +1,5 @@
+import { makeDiarizedTranscriptsPath } from 'utilities/transcript';
+
+it('renders homepage unchanged', () => {
```

Comment:
Exciting! Looks like it uses the same test structure as Jasmine. Gonna have to read up on how it hooks into React/Next components.